packages/neon/neon/lib/src/utils/relative_time.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:meta/meta.dart';
+
+@internal
+extension RelativeTimeFormat on DateTime {
+  /// Format the relative time between this and [to].
+  ///
+  /// If unspecified [DateTime.now] will be used.
+  String formatRelative([final DateTime? to]) {
+    final now = to ?? DateTime.now();
+    var diff = now.difference(toLocal());
+    final text = StringBuffer();
+
+    // Sometimes something can be messed up...
+    if (diff.isNegative) {
+      if (diff.inMinutes >= 1) {
+        text.write('-');
+      }
+      diff = Duration(microseconds: -diff.inMicroseconds);
+    }
+
+    if (diff.inMinutes < 1) {
+      text.write('now');
```

Comment:
We should add localizations for this and the abbreviations below

Reply:
Sadly after nearly a decade intl still doesn't support duration formatting :(
IIRC there are requests dating back to 2015.

But in this case we should probably use some package. A qick search found `duration` wich does support some languages https://github.com/desktop-dart/duration/tree/master/lib/src/locale. Should we rather go this route or build our own?

Reply:
If you find the quality of the package is good then go for it

Reply:
The duration package looks good!